src/test/ripple-libpp_demo.cpp

```diff
@@ -178,7 +171,189 @@ int main (int argc, char** argv)
     auto const allPass = std::all_of(passes.begin(), passes.end(),
         [](auto p) { return p; });
     assert(allPass);
-    std::cout << (allPass ? "All checks pass.\n" : "Some checks fail.\n");
+    std::cout << (allPass ?
+        "All single signing checks pass.\n" :
+        "Some single signing checks fail.\n");
+
+    return allPass;
+}
+
+//------------------------------------------------------------------------------
+
+// Demonstrate multisigning.
+
+// Helper function that throws if for some reason we can't create a seed.
+ripple::Seed getSeed (std::string const& seedText)
+{
+    // WARNING!
+    // Never use ripple::parseGenericSeed() for secure code.  Call
+    // ripple::randomSeed() instead, since it is cryptographically secure.
+    boost::optional<ripple::Seed> const possibleSeed {
+        ripple::parseGenericSeed (seedText)};
+
+    if (! possibleSeed)
+        throw std::runtime_error (
+            std::string ("Could not generate seed from: ") + seedText);
+
+    return *possibleSeed;
+}
+
+// Holds identifying information for an account.
+class Credentials
+{
+    std::string const name_;
+    ripple::KeyType const keyType_;
+    ripple::Seed const seed_;
+    std::pair<ripple::PublicKey, ripple::SecretKey> const keys_;
+    ripple::AccountID const id_;
```

Comment:
Since these are all `const`, why not leave them `public` and remove the getters?